root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard.cs

```diff
@@ -0,0 +1,797 @@
+                // 通らない予定
+            }
+
+            // 分離レベル（iso）をメンバ変数に保存
+            _iso = iso;
+        }
+
+        #endregion
+
+        #region 終了
+
+        /// <summary>トランザクションのコミット</summary>
+        /// <remarks>必要に応じて利用する。</remarks>
+        public override void CommitTransaction()
+        {
+            // Txオブジェクトの存在チェック
+            LogIF.DebugLog("ACCESS", "Transaction not supported (CommitTransaction)");
+        }
+
+        /// <summary>トランザクションのロールバック</summary>
+        /// <remarks>必要に応じて利用する。</remarks>
+        public override void RollbackTransaction()
+        {
+            //// Txオブジェクトの存在チェック
+            LogIF.DebugLog("ACCESS", "Transaction not supported (RollbackTransaction)");
+        }
+
+        #endregion
+
+        #endregion
+
+        #region SQLの作成
+
+        # region SetSql系
+
+        /// <summary>SQL文を記述したファイルへのパスを設定して、Commandオブジェクトを生成。</summary>
+        /// <param name="sqlFilePath">SQL文を記述したファイルへのパス</param>
+        /// <remarks>通常、Ｄａｏ経由で利用する。</remarks>
+        public override void SetSqlByFile(string sqlFilePath)
+        {
+            // CommandTypeをTextとしてCommandオブジェクトを生成する
+            this.SetSqlByFile(sqlFilePath, CommandType.Text);
+        }
+
+        /// <summary>SQL文を記述したファイルへのパスとCommandTypeを設定して、Commandオブジェクトを生成。</summary>
+        /// <param name="sqlFilePath">SQL文を記述したファイルへのパス</param>
+        /// <param name="commandType">コマンドの種類</param>
+        /// <remarks>通常、Ｄａｏ経由で利用する。</remarks>
+        public override void SetSqlByFile(string sqlFilePath, CommandType commandType)
+        {
+            // ファイルから、実行するSQL文を読み込む
+            string commandText = this.Load(sqlFilePath);
+
+            // SQL文を指定してCommandオブジェクトを生成する
+            this.SetSqlByCommand(commandText, commandType);
+        }
+
+        /// <summary>SQL文を設定して、Commandオブジェクトを生成。</summary>
+        /// <param name="commandText">実行するSQL文</param>
+        /// <remarks>通常、Ｄａｏ経由で利用する。</remarks>
+        public override void SetSqlByCommand(string commandText)
+        {
+            // CommandTypeをTextとしてCommandオブジェクトを生成する
+            this.SetSqlByCommand(commandText, CommandType.Text);
+        }
+
+        /// <summary>SQL文とCommandTypeを設定して、Commandオブジェクトを生成。</summary>
+        /// <param name="commandText">実行するSQL文</param>
+        /// <param name="commandType">コマンドの種類</param>
+        /// <remarks>通常、Ｄａｏ経由で利用する。</remarks>
+        public override void SetSqlByCommand(string commandText, CommandType commandType)
+        {
+            // コマンド テキストが、動的パラメタライズド クエリであるか確認する。
+            // 結果は、プロパティに保存される。
+            this.CheckCommandText(commandText);
+
+            // Commandオブジェクトを生成する
+            this._cmd = this._cnn.CreateCommand();
+            this._cmd.CommandText = commandText;
+            this._cmd.CommandType = commandType;
+
+            // システム共通のCommandTimeout値を設定する。
+            this.SetCommandTimeout(this._cmd); // #x-この行
+        }
+
+        # endregion
+
+        #region SetParameter系
```

Comment:
@daisukenishino 

"Set" and "Get" are not same "lineage".
If you want to summarize "SetParameter" and "GetParameter" in same region, you should name this region as follows.

```c#
#region SetParameter / GetParameter
```

or

```c#
#region Parameter系
```